hetmech/test_degree_weight.py

```diff
@@ -19,14 +18,16 @@ def test_disease_gene_example_dwwc():
 
     # Compute GiGaD path count and DWWC matrices
```

Comment:
I agree the name should be changed to dwwc. This will happen with https://github.com/greenelab/hetmech/pull/40/files#diff-e2d55ecf6a3953dc596c0bdc7ba2792e